app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,80 @@
+body {
```

Comment:
besides oswaldo's comments, I recommend you to follow the approach that your partners are using in this [Pull request](https://github.com/BrightCoders-Proyectos/control-almacen/pull/21), here they added the `variables` scss file to add color-variables as Oswaldo mentioned. Is a very useful style-fles-sass structure. :P